### PR TITLE
Lots of formatting fixes to do with buttons

### PR DESCRIPTION
**Pull request by mstanleyjones**
- Change all the dark blue buttons to outline buttons to match the current theme

- The outline buttons need some padding at the bottom or they will crush into the next header

- A single button doesn't need to be inside a table so I removed tables where unnecessary

- A few tables were relying on `<br />` within a `<td>`. I just added another `<tr>` and put the second line's content there.

- Updated the examples of buttons in `test.md`.

This doesn't address a few things for follow-up work:

- We really need to figure out how to use cards for horizontal content so we can avoid having two-column 50% tables.
- I didn't change all of the HTML links into Markdown format, though I did change some.
- Probably in mobile, this kind of button should change from `inline-block` to `inline` and stack vertically.

@FrenchBen @nathanleclaire PTAL at D4A
@londoncalling PTAL at D4M, D4W, Toolbox
@johndmulhausen @jsouth PTAL overall

### CODE REVIEW

File: docker-for-aws/index.md [hunk 1]
```diff
@@ -40,13 +40,10 @@ using CloudFormation. For more about stable and edge channels, see the
   </td>
   <td width="50%">
   {{aws_blue_edge}}
+  {{aws_blue_vpc_edge}}
   </td>
   </tr>
   <tr valign="top">
```
Comment (FrenchBen) [inline docker-for-aws/index.md]: this `<tr>` and following `</td></tr>`

**Comment by nathanleclaire** [comment]
Don't know that much about the CSS in docs but LGTM

File: docker-for-aws/index.md (hunk 1 above)
Comment (mstanleyjones) [inline docker-for-aws/index.md]: The `<tr>` has two `<td>` in it just like it's supposed to. I think it's right. Both buttons are in one `<td>`.

**Comment by johndmulhausen** [comment]
LGTM as well :)

**Comment by mstanleyjones** [comment]
Merging for now. 👀

**Pull request merged**